tools/osbuilder/image-builder/image_builder.sh

```diff
@@ -405,6 +411,17 @@ create_disk() {
 	OK "Partitions created"
 }
 
+setup_selinux_agent() {
+	local mount_dir="$1"
+	# need to check the distro since policies vary
+	if [ -f ${mount_dir}/etc/os-release ]; then
+		local DISTRO=`sed -n 's/^NAME=\(.*\)$/\1/p' < ${mount_dir}/etc/os-release`
+		if [ "${DISTRO}" == "\"Ubuntu\"" ]; then
+			chroot "${mount_dir}" /usr/bin/chcon "system_u:system_r:dockerd_t:s0" /usr/bin/kata-agent
```

Comment:
Hmm. Why `dockerd_t` instead of `container_t`?

Reply:
Nevermind, the agent should be `dockerd_t`, indeed.

Reply:
Thank you for taking a look. The fact the question was raised suggests a comment should be added to provide the information directly.